client/nginx.conf.docker

```diff
@@ -0,0 +1,49 @@
+worker_processes  8;
+
+error_log  /dev/stderr warn;
+pid        /var/run/nginx.pid;
+
+events {
+    worker_connections  4096;
+}
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /dev/stdout  main;
+
+    sendfile        on;
```

Comment:
The indentation for values in this file is completely arbitrary. I'd either align it in two columns, or make everything use single spaces (like `key value;`). I'm leaning towards the latter solution.